src/lightning/__setup__.py

```diff
@@ -0,0 +1,103 @@
+import glob
+import os.path
+from importlib.util import module_from_spec, spec_from_file_location
+from itertools import chain
+from types import ModuleType
+from typing import Any, Dict
+
+from setuptools import find_packages
+
+_PROJECT_ROOT = "."
+_SOURCE_ROOT = os.path.join(_PROJECT_ROOT, "src")
+_PACKAGE_ROOT = os.path.join(_SOURCE_ROOT, "lightning")
+
+
+def _load_py_module(name: str, location: str) -> ModuleType:
```

Comment:
Isn't this already in setup_tools?

Reply:
yes but you need it to be able to load `setup_tools` :)